src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,26 @@
-export const TodoInfo = () => {};
+import ToDo from '../../types/ToDo';
+import { UserInfo } from '../UserInfo';
+import cn from 'classnames';
+import usersFromServer from '../../api/users';
```

Comment:
[REACT KNOWLEDGE] - Don't render the component if the property that you pass to the component has `null` or `undefined` value. In this case, you are providing an empty string as a fallback for `foundedUser` which is not a valid user object. It would be better to provide `null` as a fallback and check for the truthiness of `foundedUser` before rendering `UserInfo`.